### PR TITLE
[OneExplorer] Show chrome trace files 

### PR DESCRIPTION
**Pull request by dayo09**
This commit shows chrome trace files on OneExplorer.

ONE-vscode-DCO-1.0-Signed-off-by: dayo09 <dayoung.lee@samsung.com>

---

From #1055
Waiting for #1058

### CODE REVIEW

File: src/OneExplorer/ConfigObject.ts [hunk 1]
```diff
@@ -412,6 +447,15 @@ export class ConfigObj {
       })
     });
 
+    locatorRunner.register({
+      artifactAttr: {ext: '.json', icon: vscode.ThemeIcon.File, canHide: true},
+      locator: new Locator(
+          (value: string) => {
+            return LocatorRunner.searchWithCommandOption(value, '--save-chrome-trace', '.json');
```
Comment (hyunsik-yoon) [inline src/OneExplorer/ConfigObject.ts:454]: `--save-chrome-trace`
- I wonder if this option is `onecc` option or option of some backend tool. If this is option of backend tool, this option needs to be obtained from backend.

File: src/OneExplorer/ConfigObject.ts [hunk 2]
```diff
@@ -23,9 +23,10 @@ import {RealPath} from '../Utils/Helpers';
 import {Logger} from '../Utils/Logger';
 
 /**
- * 'Artifact' means includes
- *   - existing files to run ONE config (base model)
- *   - result files after running ONE config (derived models, )
+ * 'Artifact'
+ * The collective term is and inclusive, it includes two types of files:
+ * (1) Pre-existing files to run ONE config (base model)
+ * (2) Result files after running ONE config (derived models, )
```
Comment (hyunsik-yoon) [inline src/OneExplorer/ConfigObject.ts:29]:
```suggestion
 * (2) Result files after running ONE config (derived models)
```

File: src/OneExplorer/ConfigObject.ts (hunk 1 above)
Comment (dayo09) [reply src/OneExplorer/ConfigObject.ts:454]: Yes, I am working on that.
But, as you know, there are several backend PRs which are waiting for reviews and merge.
Therefore I decided to work on here and move these later.

**Comment by dayo09** [comment]
@hyunsik-yoon FYI, this PR includes prerequisite PR which is not reviewed/merged yet :-D

> Waiting for https://github.com/Samsung/ONE-vscode/pull/1058

File: src/OneExplorer/ConfigObject.ts (hunk 2 above)
Comment (dayo09) [reply src/OneExplorer/ConfigObject.ts]: @hyunsik-yoon Actually, this change is part of #1058 PR. PTAL :-D
https://github.com/Samsung/ONE-vscode/pull/1058/files#r924070496

File: src/OneExplorer/ConfigObject.ts (hunk 1 above)
Comment (hyunsik-yoon) [reply src/OneExplorer/ConfigObject.ts:455]: Could you write a `TODO` from the next time so that review does not review such code? Thanks.

**Review by hyunsik-yoon — APPROVED** [review]

File: src/OneExplorer/ConfigObject.ts (hunk 1 above)
Comment (dayo09) [reply src/OneExplorer/ConfigObject.ts:455]: Okay :-D

**Review by llFreetimell — APPROVED** [review]

**Pull request merged**